geckodriver.log

```diff
@@ -0,0 +1 @@
+1624633224198	geckodriver	INFO	Listening on 127.0.0.1:54777
```

Comment:
Would be good to delete this file and add it to your `.gitignore`

Reply:
Revoked the token and created a new one.    geckodriver.log file has been added to .gitgnore file

Reply:
Done